manage.py

```diff
@@ -169,6 +178,11 @@
             'mti_pluginapp': 'cms.test_utils.project.mti_pluginapp.migrations',
         }
 
+    if os.environ.get('AUTH_USER_MODEL', False):
+        custom_user_app = 'cms.test_utils.project.' + os.environ.get('AUTH_USER_MODEL', False).split('.')[0]
+        INSTALLED_APPS.insert(INSTALLED_APPS.index('cms'), custom_user_app)
```

Comment:
Yes, it is: http://django-cms.readthedocs.org/en/develop/reference/configuration.html#custom-user-requirements